File: CS/EditorsDemo.Wpf/Modules/LookUpEditServerMode/PersonsDataContext.cs
using DevExpress.Xpf.DemoBase;
using System.Data.Entity;
using System.IO;

namespace GridDemo {
    public class PersonsDataContext : DbContext {
        public static readonly string FileName = Path.GetFullPath("PersonData.db");
        public PersonsDataContext()
            : base(SQLiteDataBaseGenerator.CreateConnection(LookUpInstantFeedbackModeViewModel.DatabaseInfo), true) {
        }
        public DbSet<Person> People { get; set; }
    }
    public class Person {
        public int Id { get; set; }
        public string FullName { get; set; }
        public string Company { get; set; }
        public string JobTitle { get; set; }
        public string City { get; set; }
        public string Address { get; set; }
        public string Phone { get; set; }
        public string Email { get; set; }
    }
}

File: VB/GridDemo.Wpf/Modules/LookUpEditServerMode/PersonsDataContext.vb
Imports DevExpress.Xpf.DemoBase
Imports System.Data.Entity
Imports System.IO

Namespace GridDemo
    Public Class PersonsDataContext
        Inherits DbContext

        Public Shared ReadOnly FileName As String = Path.GetFullPath("PersonData.db")
        Public Sub New()
            MyBase.New(SQLiteDataBaseGenerator.CreateConnection(LookUpInstantFeedbackModeViewModel.DatabaseInfo), True)
        End Sub
        Public Property People() As DbSet(Of Person)
    End Class
    Public Class Person
        Public Property Id() As Integer
        Public Property FullName() As String
        Public Property Company() As String
        Public Property JobTitle() As String
        Public Property City() As String
        Public Property Address() As String
        Public Property Phone() As String
        Public Property Email() As String
    End Class
End Namespace
